PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/CameraScript/RootCamera.rbxmx

```diff
@@ -169,6 +153,36 @@ do
 			OnCharacterAdded(character)
 		end)
 	end
+	
+	-- Check for changes in ViewportSize to decide if PortraitMode
+	local CameraChangedConn = nil
+	local function onWorkspaceChanged(property)
```

Comment:
I could have sworn I replied to your comments and submitted another commit! Sorry, let me see where I left this...